ITE/7/README.adoc

```diff
@@ -0,0 +1,255 @@
+= ITE-7: Signing & Verification With X509
+:source-highlighter: pygments
+:toc: preamble
+:toclevels: 2
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 7
+
+| Title
+| Signing & Verification With X509
+
+| Sponsor
+| link:https://github.com/mikhailswift[Mikhail Swift]
+
+| Status
+| Draft
+
+| Type
+| Standards
+
+| Created
+| 2021-07-01
+
+|===
+
+
+[[abstract]]
+== Abstract
+
+This ITE proposes adding to the in-toto specification to support signing and
+verifying link metadata with key & X509 certificate pairs.  This ITE details the
+required changes to the current layout and link specifications as well as the
+signing and verification processes.
+
+[[specification]]
+== Specification
+
+This ITE proposes modifying the existing in-toto layout and link metadata specs
+to support verifying functionaries with a chain of trust to a known root
+certificate authority.
+
+Modifications to the layout include the addition of a `rootcas` and
+`intermediatecas` field.  These fields allow the creator of the layout to
+specify known trusted certificate authorities that a functionary's signing key
+must belong to.
+
+Additionally a `cert_constraints` field is added to steps within a layout.  This
+field allows the layout creator to limit the keys a functionary by some
+attributes of the key's certificate, ensuring the key has a specific common
+name or URI, for instance.
+
+[[authorities]]
+=== Certificate Authorities
+
+The `intermediatecas` and `rootcas` fields will have the same shape as the
+existing `keys` field: an object that with string indexes that are equal to the
+key's ID.  The keyval's public key will be expected to be a valid X509
+certificate.
```

Comment:
PEM encoded? or is that up to the implementer and outside of the scope of this Proposal